src/types.ts

```diff
@@ -1251,12 +1253,24 @@ export type AppSettings = {
   disable_auth_checks?: boolean;
   disable_permissions_checks?: boolean;
   enforce_unique_usernames?: 'no' | 'app' | 'team';
+  file_upload_config?: {
+    allowed_file_extensions?: string[];
+    allowed_mime_types?: string[];
```

Comment:
Just to confirm, do we have some exhaustive list of all these types/extensions? @miagilepner

Reply:
We don't. We could make an exhaustive list (e.g. for the media types the full list is here: https://www.iana.org/assignments/media-types/media-types.xhtml) and do validation, if you think this should be prioritized.

Reply:
Not necessary tbh. If backend is prepared to handle any wierd type ... then its totally fine!!

Reply:
let's put a comment in here with the link to the accepted types